packages/grid/src/internal/TableRow.css

```diff
@@ -0,0 +1,48 @@
+.uitkGridTableRow {
+  height: var(--uitkGrid-rowHeight);
+  /* Row border doesn't render properly in Chrome on Windows. Applying borders to cells */
+  --row-border-color: var(--uitkGrid-rowDivider-color);
+}
+
+.uitkGridTableRow th {
+  border-style: solid;
+  border-color: var(--row-border-color);
+  border-width: 1px 0 0 0;
+}
+
+.uitkGridTableRow-selected + .uitkGridTableRow {
+  --row-border-color: var(--uitkGrid-row-borderColor-selected);
+}
+
+.uitkGridTableRow-divided {
+  --row-border-color: var(--uitkGrid-rowDivider-color-divided);
+}
+
+.uitkGridTableRow-zebra {
+  background: var(--uitkGrid-zebraColor);
+}
+
+.uitkGridTableRow-hover {
+  background: var(--uitkGrid-row-backgroundColor-hover);
+}
+
+.uitkGridTableRow-selected {
+  background: var(--uitkGrid-row-backgroundColor-selected);
+  --row-border-color: var(--uitkGrid-row-borderColor-selected);
```

Comment:
needs correct prefix and css attr should be kebab case (you could check this by locally running the lint against grid package)

Reply:
You mean camel case? e.g. "--grid-row-borderColor"?

Reply:
yes sorry :)